ci/pyapp.py

```diff
@@ -0,0 +1,15 @@
+import cryptography.fernet
```

Comment:
this is copied from https://github.com/jamesabel/spafit/pull/2

It is a good smoketest for basic cryptography.